documentation/announcement.md

```diff
@@ -0,0 +1,44 @@
+# Announcing canonical 0.6
```

Comment:
The announcement was deleted from the repo (and added to the wiki), no need to reintroduce it.
Be sure you're in sync with the latest master of canonical, just in case.

Basically the whole `documentation` folder shouldn't be here.